test/blackbox-tests/test-cases/cmdline/profile.t

```diff
@@ -1,9 +1,9 @@
 # Bug #4632
 
   $ dune build -p
-  dune build: option `-p' needs an argument
```

Comment:
error messages do not output the command name now. I haven't tried to see if this is something that we can fix but this looks like a regression with how the messages look

Reply:
Did you ask upstream about his?

Reply:
Good point. I started working a patch to see if this can fit in cmdliner.

Reply:
I patched locally and will submit the PR upstream.